### PR TITLE
Lowercase json to be more compliant with javascript coding standards

### PR DESCRIPTION
**Pull request by marcofranssen**
To have all the json start with a lowercase I added the annotations on the models. This way the json responses on rest api are more aligned with REST Standards + the fact someone can use the json objects in javascript in such way the json is still compliant with javascript coding standards.

### CODE REVIEW

File: config/config.go [hunk 1]
```diff
@@ -54,34 +54,38 @@ var (
...
 
 // Post holds the bot configuration data
 type Post struct {
-	Data Config `json:"Data"`
+	Data Config `json:"data"`
 }
 
 // CurrencyPairFormatConfig stores the users preferred currency pair display
 type CurrencyPairFormatConfig struct {
 	Uppercase bool
```
Comment (thrasher-) [inline config/config.go]: This can be made lowercase too

File: config/config.go [hunk 2]
```diff
@@ -90,130 +94,130 @@ type CurrencyPairFormatConfig struct {
 type Config struct {
 	Name              string
```
Comment (thrasher-) [inline config/config.go]: These two settings (Name and EncryptConfig) can be made lowercase.

Comment (marcofranssen) [reply config/config.go]: good catch... missed these indeed

**Comment by marcofranssen** [comment]
Updated config_example and resolved conflicts.

**Pull request merged**